package.json

```diff
@@ -11,27 +11,27 @@
   "homepage": ".",
   "dependencies": {
     "@codemirror/lang-java": "6.0.1",
-    "@codemirror/lang-javascript": "6.1.1",
-    "@codemirror/lang-python": "6.1.0",
+    "@codemirror/lang-javascript": "6.1.2",
+    "@codemirror/lang-python": "6.1.1",
     "@emotion/react": "11.10.5",
     "@emotion/styled": "11.10.5",
     "@fortawesome/fontawesome-svg-core": "6.2.1",
     "@fortawesome/free-regular-svg-icons": "6.2.1",
     "@fortawesome/free-solid-svg-icons": "6.2.1",
-    "@fortawesome/react-fontawesome": "latest",
-    "@graasp/apps-query-client": "github:graasp/graasp-apps-query-client#v0",
+    "@fortawesome/react-fontawesome": "0.2.0",
+    "@graasp/apps-query-client": "^0.4.1",
```

Comment:
Should we use the caret notation or not? It would make sense to pin for consistency, but we'll have to add renovatebot to ensure that they are brought up-to-date

This also applies to all our other graasp deps

Reply:
Right I think we should pin, will update.

👍 for Renovatebot